core/src/test/java/org/infinispan/filter/CompositeKeyValueFilterConverter.java

```diff
@@ -14,6 +19,7 @@
  * @author wburns
  * @since 7.0
  */
+@SerializeWith(CompositeKeyValueFilterConverter.Ext.class)
```

Comment:
> why using the annotation instead of creating an AdvancedExternalizer like in the other classes?

Probably cos it wasn't a class that's needed by core per se. I mean, the user can use it for doing dealing with composite keys but it felt like an edge case rather than a common case. Then again, the rest of of helper classes, e.g. `KeyFilterAsKeyValueFilter` have advanced externalizers pre-registered, so I'll change that accordingly.